File: seminar/src/app/classes/Schedule.ts
import { Teacher } from 'src/app/classes/Teacher';
import { Class } from 'src/app/classes/class';
import { subject } from 'src/app/classes/subject';


export class Schedule {
    constructor(
        // public Code_schedule:number=0,
        public Hour_in_dey: number = 0,
        public Clss: Class = new Class(),
        public Day: number = 0,
        public SubjectIn: subject = new subject(),
        public teacher: Teacher = new Teacher()
    ) { }
}

File: Backend/common/Schedule.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace common
{
    public class Schedule
    {
        private int code_schedule;

        public int Code_schedule
        {
            get { return code_schedule; }
            set { code_schedule = value; }
        }

        private int? hour_in_dey;

        public int? Hour_in_dey
        {
            get { return hour_in_dey; }
            set { hour_in_dey = value; }
        }
        private int? clss;

        public int? Clss
        {
            get { return clss; }
            set { clss = value; }
        }
        private int? day;

        public int? Day
        {
            get { return day; }
            set { day = value; }
        }
        private int? subjectIn;

        public int? SubjectIn
        {
            get { return subjectIn; }
            set { subjectIn = value; }
        }
        private int? teacher;

        public int? Teacher
        {
            get { return Teacher; }
            set { Teacher = value; }
        }
        public bool? su;
        public bool? Su
        {
            get { return su; }
            set { su = value; }
        }
 
        //public static implicit operator Schedule(global::dal.schedule v)
        //{
        //    throw new NotImplementedException();
        //}
    }
}

